files/en-us/web/javascript/reference/global_objects/array/find/index.html

```diff
@@ -48,7 +48,7 @@ <h3 id="Parameters">Parameters</h3>
 <h3 id="Return_value">Return value</h3>
 
 <p>The <strong>value</strong> of the <strong>first element</strong> in the array that satisfies the provided testing function. Otherwise, {{jsxref("undefined")}} is returned.</p>
-
+<p class="warning">Index zero will be interpreted as a falsy value in conditional statements.</p>
```

Comment:
```suggestion
<p class="warning">Index <code>0</code> will be interpreted as a {{Glossary("Falsy")}} value in conditional statements.</p>

```
Let’s include a newline before the heading.

Reply:
awesome- thanks for fixing up.